pkg/gcv/cf/cf.go

```diff
@@ -139,10 +148,21 @@ func staticCompile(auditScript string, dependencyCode map[string]string, templat
 }
 
 // Reset the user provided data, preserving the constraint and template information.
-func (cf *ConstraintFramework) Reset() {
+func (cf *ConstraintFramework) Reset(ctx context.Context) error {
 	// Clear input data
 	// This is provided as a param in audit
 	cf.userInputData = []interface{}{}
+	return cf.setStoreInventory(ctx)
+}
+
+func (cf *ConstraintFramework) Review(ctx context.Context, objJSON interface{}) ([]*validator.Violation, error) {
```

Comment:
We should document this method, particularly given the ambiguous "objJSON" arg (which could probably have a better name).

Reply:
Added docstring.  As for the name, I was using the same one that the real [constraint framework uses](https://github.com/open-policy-agent/frameworks/blob/master/constraint/pkg/client/client.go#L504) (I suppose we could name it different, but validator will eventually always call Review at some point).